regression/regression_param.sh

```diff
@@ -87,7 +87,8 @@ case $regtest in
     rrfs_3denvar_glbens)
 
         if [[ "$machine" = "Hera" ]]; then
-           topts[1]="0:15:00" ; popts[1]="20/1/"  ; ropts[1]="/1"
+#cltorg           topts[1]="0:15:00" ; popts[1]="20/1/"  ; ropts[1]="/1"
```

Comment:
No need to retain commented out lines.  Version control tracks changes.  `git diff` of various hashes will show exactly how a file has changed.